tests/test_startup.py

```diff
@@ -0,0 +1,14 @@
+import virtool.startup
+
+
+async def test_init_faker(snapshot, spawn_client):
+    client = await spawn_client(authorize=True)
+
+    app = client.app
+
+    await virtool.startup.init_faker(app)
+
+    faker = app["faker"]
+
+    assert faker
```

Comment:
Not sure what the point of this is. Should be checking with `isinstance` or something a bit more informative.